presto-hive/src/main/java/io/prestosql/plugin/hive/authentication/AuthenticationModules.java

```diff
@@ -78,11 +77,8 @@ public static Module noHdfsAuthenticationModule()
     public static Module simpleImpersonatingHdfsAuthenticationModule()
     {
         return binder -> {
-            binder.bind(Key.get(HadoopAuthentication.class, ForHdfs.class))
-                    .to(SimpleHadoopAuthentication.class);
-            binder.bind(HdfsAuthentication.class)
-                    .to(ImpersonatingHdfsAuthentication.class)
-                    .in(SINGLETON);
+            binder.bind(HadoopAuthentication.class).annotatedWith(ForHdfs.class).to(SimpleHadoopAuthentication.class);
```

Comment:
I would restore previous code formatting. Here and elsewhere.

Reply:
we usually write them on one line elsewhere.
 (btw otherwise this would be 3 lines)

Reply:
ok, "usually" is an overstatement. "often" is more proper. It's actually a pretty even mix today.

anyway, i feel like cascading this doesn't increase readability.

it does help when we bind `instance` (not class) and the binding stanza would be too long for one line.
Here it's not.